test/test_torch.py

```diff
@@ -8850,6 +8853,21 @@ def test_from_buffer(self):
         self.assertEqual(floats.size(), 1)
         self.assertEqual(floats[0], 2.25)
 
+        f = bytearray([0x00, 0x01, 0x02, 0x03, 0x04, 0x05, 0x10, 0x40])
+        bools = torch.BoolStorage.from_buffer(f, 'big')
```

Comment:
You are right, we dont need to pass it but as from_buffer is auto generated from a template, it expects two values to be passed. Its even smart enough to give an error in case some unexpected value is being passed instead if 'big'. Error: "ValueError: invalid byte_order '' (expected 'big', 'little', or 'native')'